packages/dds/merge-tree/src/segmentInfos.ts

```diff
@@ -325,32 +245,21 @@ export const isMoved = (segmentLike: unknown): segmentLike is IMoveInfo =>
  * When this happens, the segment is only ever visible to the client that inserted the segment
  * (and only until that client has seen the obliterate which removed their segment).
  */
-export function wasMovedOnInsert(segment: IInsertionInfo & ISegmentPrivate): boolean {
-	const moveInfo = toMoveInfo(segment);
-	const movedSeq = moveInfo?.movedSeq;
+export function wasMovedOnInsert(segment: IHasInsertionInfo & ISegmentPrivate): boolean {
```

Comment:
related to slice vs set, should we move away from "move"

Reply:
Yes, might as well now that the in-memory bits are decoupled.